src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -283,16 +298,36 @@ void ThreadStateBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assembler,
       });
 }
 
+std::optional<ThreadStateSliceInfo> ThreadStateBar::FindSliceFromScreenCoords(float x,
+                                                                              float y) const {
+  Vec2i screen_pos = Vec2i(static_cast<int>(x), static_cast<int>(y));
+  float world_pos_x = viewport_->ScreenToWorld(screen_pos)[0];
```

Comment:
This is wrong. Is working because in viewport screen_size and world_size has the same value but we shouldn't rely on that.